### DISCUSSION TITLE
Allegati al caso uso 1324 (Trascr_010)

### DISCUSSION THREAD

**Discussion by biadanpad** [italia/ansc#1200] [Supporto servizi cooperativi] [answered]
Buonasera, 
nel mapping del caso d'uso in oggetto (Trascrizione richiesta dall’interessato della sentenza straniera di disconoscimento di paternità) viene richiesto come allegato obbligatorio l'atto di nascita dell'interessato.
Mi viene il dubbio che l'allegato corretto sia invece l'atto di nascita dell'intestatario inquanto l'interessato (il richiedente) potrebbe non avere tale atto.

Potreste cortesemente verificare?

**Comment by biadanpad** [comment]
Anche il caso d'uso 1325 richiede come allegato obbligatorio l'atto di nascita dell'interessato, ma qui la richiesta non avviene da un interessato: arriva da un'autorità diplomatica o consolare

**Comment by acolosimosogei** [comment]
Buongiorno, 

nei casi d'uso 1324 e 1325 dovrebbero comparire come allegati obbligatori gli atti di nascita dell'intestatario. 
Deve esserci stato un errore che procederemo a risolvere quanto prima,

Grazie per la segnalazione.

Assistenza ANSC

**Comment by biadanpad** [comment]
Ho appena notato che nel caso d'uso 1363 viene richiesto come allegato obbligatorio la "Copia integrale dell'atto di nascita del dichiarante".
E' giusto oppure anche in questo caso dovrebbe essere sostituito con l'atto di nascita dell'intestatario?

**Comment by asqueglia** [comment, marked as answer]
Con il rilascio della versione 1.36.0, la segnalazione è stata corretta
